client/config/index.js

```diff
@@ -20,6 +20,7 @@ for (let chainId in contracts) {
 const supportedId = {};
 supportedId[56] = "BSC";
 supportedId[97] = "BSC Testnet";
+supportedId[43113] = "Fuji";
 supportedId[1337] = "localhost";
```

Comment:
We should support only 56 in production. The testnet should be supported only when connected to localhost.
(See how we set it in config/index.js for the Staking app)